evaluate/metrics/assessment.go

```diff
@@ -0,0 +1,40 @@
+package metrics
+
+// AssessmentKey is a description for a numerical assessment value.
+type AssessmentKey string
+
+// allAssessmentKeys holds all assessment keys.
+var allAssessmentKeys []AssessmentKey
+
+func registerAssessmentKey(key string) AssessmentKey {
+	assessment := AssessmentKey(key)
+	allAssessmentKeys = append(allAssessmentKeys, assessment)
+
+	return assessment
+}
+
+var (
+	// AssessmentKeyNoExplanations means that a model did not produce additional explanations.
+	AssessmentKeyNoExplanations = registerAssessmentKey("no-explanations")
```

Comment:
- -> `AssessmentKeyNoAdditionalExplanations`
- -> `no-additional-explanations`

because, it is about the "additional". if we would have a prompt where we require explanations... that key would kind of be the same

Reply:
Now that i a have seen the test cases. i am wondering if this should be something like "More content than required" or something like that. What we are looking for is if there is more than just the code fence. If it is more than we asked for.  So maybe "AnswerContainingMoreThanWeAskedFor" or something?

Reply:
Then maybe `AssessmentKeyExcessResponse`?

Reply:
Or then `AssessmentKeyNoExcessResponse` as we reward when there is no additional stuff.